packages/extension/src/pages/main/index.tsx

```diff
@@ -66,22 +66,22 @@ export const MainPage: FunctionComponent = observer(() => {
     }
   }, [chainStore, confirm, chainStore.isInitializing, currentChainId, intl]);
 
-  const accountInfo = accountStore.getAccount(chainStore.current.chainId);
+  // const accountInfo = accountStore.getAccount(chainStore.current.chainId);
 
-  const queryBalances = queriesStore
-    .get(chainStore.current.chainId)
-    .queryBalances.getQueryBech32Address(accountInfo.bech32Address);
+  // const queryBalances = queriesStore
+  //   .get(chainStore.current.chainId)
+  //   .queryBalances.getQueryBech32Address(accountInfo.bech32Address);
 
-  const tokens = queryBalances.unstakables.filter((bal) => {
-    // Temporary implementation for trimming the 0 balanced native tokens.
-    // TODO: Remove this part.
-    if (new DenomHelper(bal.currency.coinMinimalDenom).type === "native") {
-      return bal.balance.toDec().gt(new Dec("0"));
-    }
-    return true;
-  });
+  // const tokens = queryBalances.unstakables.filter((bal) => {
```

Comment:
Why is the following section commented

Reply:
Its the old logic for checking if the accoun has tokens for not. I thight we might need it later when we get data so commented for now